### PR TITLE
chore(refact): change entry point structre

### PR DESCRIPTION
**Pull request by zerubeus**
# بسم الله الرحمن الرحيم

## What's new in this PR?

fixes

لله الحمد والمنة

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,6 +73,7 @@
     "react-i18next": "^11.3.4",
     "react-icons": "^2.2.7",
     "react-select": "^3.1.0",
+    "react-teleporter": "^2.1.1",
```
Comment (abumalick) [inline package.json:83]: interesting library

File: src/services/auth.ts [hunk 2]
```diff
@@ -41,6 +41,12 @@ firebase.initializeApp({
 
 const googleProvider = new firebase.auth.GoogleAuthProvider()
 
+const bearer = (idToken: string) => ({
```
Comment (abumalick) [inline src/services/auth.ts]: I would have called it `createHeaders` or something like this.

**Review by abumalick — APPROVED** [review]
LGTM 👍

File: src/services/auth.ts (hunk 2 above)
Comment (zerubeus) [reply src/services/auth.ts]: Yes createHeaders makes more sense

**Pull request merged**